docs: clarify that mixin flag should be unique (#1243)
nuxt/nuxtjs.org: content/en/guides/directory-structure/plugins.md

Global mixins can be easily added with Nuxt plugins but can cause trouble and memory leaks when not handled correctly. Whenever you add a global mixin to your application, you should use a flag to avoid registering it multiple times:

> ```js{}[plugins/my-global-mixin.js]

Reviewer: ```suggestion
```js{}[plugins/my-mixin-plugin.js]
```

Global mixins can be easily added with Nuxt plugins but can cause trouble and memory leaks when not handled correctly. Whenever you add a global mixin to your application, you should use a flag to avoid registering it multiple times:

```js{}[plugins/my-global-mixin.js]
import Vue from "vue"

> if (!Vue._myGlobalMixin) { // Define a new name for each plugin

Reviewer: ```suggestion
// Make sure to pick a unique name for the flag
// so it won't conflict with any other mixin.
if (!Vue._myGlobalMixin) {
```

Global mixins can be easily added with Nuxt plugins but can cause trouble and memory leaks when not handled correctly. Whenever you add a global mixin to your application, you should use a flag to avoid registering it multiple times:

```js{}[plugins/my-global-mixin.js]
import Vue from "vue"

// Make sure to pick a unique name for the flag
// so it won't conflict with any other mixin.

> if (!Vue._myGlobalMixin) {
>   Vue._myGlobalMixin = true

Reviewer: ```suggestion
if (!Vue.__my_mixin__) {
  Vue.__my_mixin__ = true
```
